README.rst

```diff
@@ -30,13 +30,286 @@
+The current release of the Triton Inference Server is 2.0.0 and
+corresponds to the 20.06 release of the tensorrtserver container on
+`NVIDIA GPU Cloud (NGC) <https://ngc.nvidia.com>`_. The branch for
+this release is `r20.06
+<https://github.com/NVIDIA/triton-inference-server/tree/r20.06>`_.
+
+Backwards Compatibility
+-----------------------
+
+Version 2 of Triton is beta quality, so you should expect some changes
+to the server and client protocols and APIs. Version 2 of Triton does
+not generally maintain backwards compatibility with
+version 1. Specifically, you should take the following items into
+account when transitioning from version 1 to version 2:
+
+* The Triton executables and libraries are in /opt/tritonserver. The
+  Triton executable is /opt/tritonserver/bin/tritonserver.
+
+* Some *tritonserver* command-line arguments are removed or have
+  different default behavior in version 2.
+
+  * --api-version, --http-health-port, --grpc-infer-thread-count,
+    --grpc-stream-infer-thread-count,--allow-poll-model-repository
+    and --allow-model-control are removed.
+
+  * The default for --model-control-mode is changed to *none*.
+
+* The HTTP/REST and GRPC protocols, while conceptually similar to
+  version 1, are completely changed in version 2. See the `inference
+  protocols
+  <https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/http_grpc_api.html>`_
+  section of the documentation for more information.
+
+* Python and C++ client libraries are re-implemented to match the new
+  HTTP/REST and GRPC protocols. The Python client no longer depends on
+  a C++ shared library and so should be usable on any platform that
+  supports Python. See the `client libraries
+  <https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/client_library.html>`_
+  section of the documentaion for more information.
+
+* The version 2 cmake build requires these changes:
+
+  * The cmake flag names have changed from having a TRTIS prefix to
+    having a TRITON prefix. For example, TRITON_ENABLE_TENSORRT.
+
+  * The build targets are *server*, *client* and *custom-backend* to
+    build the server, client libraries and examples, and custom
+    backend SDK, respectively.
+
+* In the Docker containers the environment variables indicating the
+  Triton version have changed to have a TRITON prefix, for example,
+  TRITON_SERVER_VERSION.
+
+Roadmap
+-------
+
+The 20.03.1 release of Triton consists of a single server/container
+that supports both the existing version 1 APIs and protocols and the
+new version 2 APIs and protocols. For version 2 the release is beta
+quality and includes the new `HTTP/REST and GRPC protocols
+<https://github.com/kubeflow/kfserving/tree/master/docs/predict-api/v2>`_
+and corresponding new C++ and Python client libraries. Version 2 also
+includes a beta release of the new server C API defined in
+tritionserver.h.
+
+The upcoming 20.06 release of Triton will include two separate server
```

Comment:
Need present tense since this is the release.
"The 20.06 release of Triton includes ..."